.github/workflows/docs.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           useRollingCache: true
           install-command: npm ci
       - name: Install dependencies (Python)
-        run: pip install -r packages/docutils/requirements.txt --break-system-packages
+        run: pip install -r packages/docutils/requirements.txt
```

Comment:
we might follow @KazuCocoa advice here by providing `BREAK_SYSTEM_PACKAGE=1` environment variable. This should work for both older and newer pip versions

Reply:
Here CI is calling pip directly, without invoking docutils, so we can't really add any flags

Reply:
weird.. wouldn't `BREAK_SYSTEM_PACKAGE=1 pip install -r packages/docutils/requirements.txt` work as expected?